alibi_detect/saving/tests/test_saving.py

```diff
@@ -259,6 +268,22 @@ def preprocess_hiddenoutput(classifier, backend):
     return preprocess_fn
 
 
+@parametrize('cfg', CFGS)
+def test_load_simple_config(cfg, tmp_path):
```

Comment:
Worth adding a docstring saying exactly what this is testing, i.e. not using the `save_detector` functionality here as that adds `[meta]` field automatically?